pages/index.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/index.css" rel="stylesheet">
+  <title>Our Group FanSite</title>
 </head>
 <body>
-  
+  <header>
+    <figure>
+      <img src="https://images.squarespace-cdn.com/content/v1/61fa82a4955b5b3d6f9f6322/989b3c57-706b-493e-a68d-5c9cce42b70e/i+love+my+pet+logo" width="120" 
+      height="100" alt="Top Image">
```

Comment:
Alt attributes should have more detailed descriptions of what the image should be as opposed to where it is placed!